### PR TITLE
adding annotation to study file and updating synth populate script (SCP-2824)

### PR DESCRIPTION
**Pull request by devonbush**
@bistline  This likely still needs work to make sure the test passes -- I'm not sure if the needed genome/annotations are populated.  But it works in development, so I thought I'd put it up so you can grab the model/populator changes.

Also, since this doesn't impact UI/UX, it might as well go on development, and then we can merge development into raw_counts_ingest

### CODE REVIEW

File: app/lib/synthetic_study_populator.rb [hunk 1]
```diff
@@ -61,35 +61,25 @@ def self.add_files(study, study_config, synthetic_study_folder, user)
...
         name: finfo['name'] ? finfo['name'] : finfo['filename'],
         upload: infile,
         use_metadata_convention: finfo['use_metadata_convention'] ? true : false,
         status: 'uploading',
-        study: study,
-        taxon_id: taxon_id
+        study: study
       }
 
+      study_file_params = study_file_params.merge(process_genomic_file_params(finfo))
```
Comment (bistline) [inline app/lib/synthetic_study_populator.rb:74]: Small nit - you can call `study_file_params.merge!(process_genomic_file_params(finfo))` instead.

File: app/lib/synthetic_study_populator.rb [hunk 2]
```diff
@@ -61,35 +61,25 @@ def self.add_files(study, study_config, synthetic_study_folder, user)
...
-        if taxon_id.nil?
-          throw "You must specify a species in the study_info.json for Expression Matrix files"
-        end
         exp_finfo_params = finfo['expression_file_info']
         if exp_finfo_params.present?
           exp_file_info = ExpressionFileInfo.new(
             is_raw_counts: exp_finfo_params['is_raw_counts'] ? true : false,
             units: exp_finfo_params['units'],
-            library_construction_protocol: exp_finfo_params['library_construction_protocol']
+            library_construction_protocol: exp_finfo_params['library_construction_protocol'],
```
Comment (bistline) [inline app/lib/synthetic_study_populator.rb:82]: Extraneous comma at the end

**Review by bistline — APPROVED** [review]
Small non-blocking styles comments.

File: app/lib/synthetic_study_populator.rb (hunk 1 above)
Comment (devonbush) [reply app/lib/synthetic_study_populator.rb:74]: oooh, nice, I hadn't used that Ruby method before

**Review by eweitz — APPROVED** [review]

**Pull request merged**